workflow-job

```diff
@@ -330,11 +376,15 @@ flows["${sha1}"] = {
                 def dUser = dockerUser();
                 def dPassword = dockerPassword();
                 sh "docker login -u ${dUser} -p ${dPassword}"
-                sh "docker tag -f ${images.james} linagora/james-project"
+                sh "docker tag -f ${images.jamesCassandra} linagora/james-project"
                 sh "docker push linagora/james-project"
 
                 sh "docker tag -f ${images.jamesJava6} linagora/james-jpa-spring"
                 sh "docker push linagora/james-jpa-spring"
+
+                sh "docker tag -f ${images.jamesJPA} linagora/james-jpa-guice"
+                sh "docker push linagora/james-guice-jpa"
+
```

Comment:
Generally, we can go faster because we can do deployment stages parrallely...

```
    parrallel("Deploying and validating james Cassandra Guice": {
        stage("Deploying James Cassandra Guice") 
        node {...}
        stage("Validating James Cassandra Guice") 
        node {...}
    }, "Deploying and validating james JPA Guice": {
        stage("Deploying James JPA Guice") 
        node {...}
        stage("Validating James JPA Guice") 
        node {...}
    }, "Deploying and validating james JPA Spring": {
        stage("Java 6 integration tests") 
        node {...}
    })
```

This will save us several minutes on integration testing...

This can be done in a new commit.